src/odemis/driver/test/xt_client_test.py

```diff
@@ -1179,5 +1179,104 @@ def test_beam_powerVA(self):
         self.scanner.power.value = init_beam_power
 
 
+class TestXTTKDetector(unittest.TestCase):
+    """
+    Test image acquisition using the XTToolkit Detector class.
+    """
```

Comment:
Aren't all these tests already done in `TestMicroscope` (now that there no special "md-detector")? 
I quickly search for differences, and couldn't see any. From a theoretical point of view, I think that the test for the XTToolKit and the XTLib should be the same, as the code is supposed to behave the same.

Reply:
Good catch, yes they're all the same. I'll remove this class.